### DISCUSSION TITLE
Analytical solution for advection diffusion reaction with a first order reaction

### DISCUSSION THREAD

**Discussion by oscarhx** [PMEAL/OpenPNM#2649] [Q&A]
Hello Professors,

Recently I received a comment said "In addition, if the authors consider a first-order reaction, then the advection-diffusion-reaction equation can be solved analytically, from which the total flux can be determined. In that case, no discretization is actually needed."
It is beyond my mathmatic ability. Is the analytical solution really exist? If so, How to solve it.

Best,

**Comment by ma-sadeghi** [comment]
Hi @oscarhx, could you add more context please. What do you mean you received a comment? You mean in OpenPNM forums? Anyway, yes there exists such a solution and you can find it probably in Crank's handbook of diffusion, or maybe googling it would be easier.

**Comment by oscarhx** [comment]
Hello Professor Sadeghi,
It's extracted from a reviewer's opinion of a denied submission.
Therefore, it doesn't matter that much.
I didnot find a direct solution to it. 
Thanks for you advice.

**Reply by ma-sadeghi** [reply]
You can use Wolfram Alpha ([source](https://www.wolframalpha.com/input?i=%7Ba*d2u%2Fdx2+%2B++b*du%2Fdx+%2B+c*u%3D+0%2C+u%280%29%3Du1%2C+u%28L%29+%3D+u2%7D)):

PS. By the way, I'm not a professor :-)

![image](https://user-images.githubusercontent.com/14086031/208152245-de57a977-d3dc-428c-8228-f7b2abacda3e.png)

**Reply by oscarhx** [reply]
hello Dr Sadeghi
Thanks, strange knowledge is increased.
